libnmstate/validator.py

```diff
@@ -34,12 +41,18 @@
 from libnmstate.schema import LinuxBridge as LB
 from libnmstate.schema import OVSBridge
 from libnmstate.schema import VXLAN
-from libnmstate.schema import Bond
 from libnmstate.error import NmstateDependencyError
 from libnmstate.error import NmstateNotImplementedError
 from libnmstate.error import NmstateValueError
 from libnmstate.iplib import is_ipv6_address
 
+SLAVE_GETTERS = {
```

Comment:
Maybe `IP_DISABLED_SLAVE_GETTERS`?

For VLAN and OVS Interface, these slaves still can have IP enabled.

Reply:
Also need a test in VLAN showing this is not impacted by new validator.

Reply:
I've changed the name and made the test